topo_processor/cli/geostore/list.py

```diff
@@ -0,0 +1,36 @@
+import click
+from linz_logger import LogLevel, get_log, set_level
+
+from topo_processor.geostore.invoke import invoke_lambda
+from topo_processor.util.time import time_in_ms
+
+
+@click.command()
+@click.option(
+    "-s",
+    "--survey",
+    required=False,
+    help="The survey to filter",
+)
+@click.option(
+    "-v",
+    "--verbose",
+    is_flag=True,
+    help="Use verbose to display debug logs",
+)
+def main(dataset_id: str, verbose: bool) -> None:
+    start_time = time_in_ms()
+    get_log().info("list_datasets_start", dataset_id=dataset_id)
```

Comment:
Is running `get_log()` for every log occurrence intentional? Would it be possible to pull out a constant/variable instead? Ditto everywhere there's more than one `get_log()` call in a single context.

Reply:
No, good point. As we do that everywhere in the code, I'll rather create another Pull Request to fix that everywhere